test/integration/gateways/pin_pay_test.exs

```diff
@@ -23,46 +24,33 @@ defmodule Gringotts.Integration.Gateways.PinpaymentsTest do
     brand: "VISA"
   }
 
-  @bad_card2 %CreditCard{
-    first_name: "Harry",
-    last_name: "Potter",
-    number: "4600000000000006",
-    year: 2019,
-    month: 12,
-    verification_code: "123",
-    brand: "VISA"
-  }
-
   @good_card %CreditCard{
     first_name: "Harry",
     last_name: "Potter",
     number: "4200000000000000",
-    year: 2019,
+    year: 2029,
     month: 12,
     verification_code: "123",
     brand: "VISA"
   }
 
   @add %Address{
-
     street1: "OBH",
     street2: "AIT",
     city: "PUNE",
-    region: "Maharashtra", 
+    region: "Maharashtra",
     country: "IN",
     postal_code: "411015",
-    phone: "8007810916",
-    
+    phone: "8007810916"
   }
 
   @opts [
-    description: "hello", 
-    email: "hi@hello.com",
-    ip_address: "1.1.1.1",
-    config: %{apiKey: "c4nxgznanW4XZUaEQhxS6g", pass: ""}
+          description: "hello",
+          email: "hi@hello.com",
+          ip_address: "1.1.1.1",
+          config: %{apiKey: "c4nxgznanW4XZUaEQhxS6g"}
+        ] ++ [address: @add]
```

Comment:
I've already pointed out in an earlier comment that you should not concat lists like this. The second list has only one element, moreover, the first list is being defined right there. You can simply move `address: @add` inside.